README.md

```diff
@@ -643,18 +644,19 @@ You can also include any of the optional configuration just like the example in
 
 ##### HuaweiCloud OBS
 
-To use HuaweiCloud OBS as an object store, you should apply for a HuaweiCloud Account to create an object storage bucket at first.
-More details: [HuaweiCloud OBS](https://support.huaweicloud.com/obs/index.html)
+To use HuaweiCloud OBS as an object store, you should apply for a HuaweiCloud Account to create an object storage bucket at first. More details: [HuaweiCloud OBS](https://support.huaweicloud.com/obs/index.html)
 
 To configure HuaweiCloud Account to use OBS as storage store you need to set these parameters in YAML format stored in a file:
 
 ```yaml mdox-exec="go run scripts/cfggen/main.go --name=cos.Config"
-type: OBS                    
-config:                      
-  bucket: ""                 
-  endpoint: ""               
-  access_key: ""             
+type: COS
```

Comment:
is this correct? It talks about Huawei OBS above and then COS?

Reply:
Let me double-check. I think running the linter or some other make action caused this.

Reply:
Thanks for catching it. I've fixed the command above that generates this doc.